src/site/paragraphs/table.drupal.liquid

```diff
@@ -0,0 +1,61 @@
+{% comment %}
+    Example data:
+    entity: {
+        "entityId": "3944",
+        "fieldTable": {
+            "tableValue": "Row 1 Column 1 Row 1 Column 2 Row 1 Column 3 Row 2 Column 1 Row 2 Column 2 Row 2 Column 3 Row 3 Column 1 Row 3 Column 2 Row 3 Column 3",
+            "value": {
+                "0": [
+                "Row 1 Column 1",
+                "Row 1 Column 2",
+                "Row 1 Column 3"
+                ],
+                "1": [
+                "Row 2 Column 1",
+                "Row 2 Column 2",
+                "Row 2 Column 3"
+                ],
+                "2": [
+                "Row 3 Column 1",
+                "Row 3 Column 2",
+                "Row 3 Column 3"
+                ],
+            }
+            "format": "rich_text",
+            "caption": "Test Table"
+        }
+    }
+{% endcomment %}
+
+
+<table class="va-table">
```

Comment:
Do we need this `va-table` class? I don't see it in the design.va.gov table example code.